enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/UniqueConstraintVerificationAcceptanceTest.scala

```diff
@@ -1,138 +0,0 @@
-/*
- * Copyright (c) 2002-2018 "Neo Technology,"
- * Network Engine for Objects in Lund AB [http://neotechnology.com]
- *
- * This file is part of Neo4j.
- *
- * Neo4j is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
- * published by the Free Software Foundation, either version 3 of the
- * License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program. If not, see <http://www.gnu.org/licenses/>.
- */
-package org.neo4j.internal.cypher.acceptance
-
-import org.neo4j.cypher.internal.compiler.v3_4.helpers.ListSupport
-import org.neo4j.cypher.{CypherExecutionException, ExecutionEngineFunSuite, QueryStatisticsTestSupport}
-import org.neo4j.kernel.api.exceptions.schema.CreateConstraintFailureException
-import org.neo4j.kernel.api.schema.SchemaDescriptorFactory
-import org.neo4j.kernel.impl.api.OperationsFacade
-
-import scala.collection.JavaConverters._
-
-class UniqueConstraintVerificationAcceptanceTest
```

Comment:
We should still have this test I feel, but change Cypher to verify that the constraint exists instead of hacking into the internals.

Reply:
migrated